### ISSUE TITLE
bluebirdCoroutines default property issue

### ISSUE THREAD

**Issue by guybedford** [babel/babel#1219]
Babel 5 is running well in jspm now.

When using the `bluebirdCoroutines` option, an issue came up recently, where the output for System looks like:

```javascript
System.register(["bluebird"], function (_export) {
  var _bluebird;

  return {
    setters: [function (_bluebird2) {
      _bluebird = _bluebird2;
    }],
    execute: function () {
      "use strict";
      // ...
    }
  };
});
```

With the above, bluebird is CommonJS so needs to be accessed from the `default` property to work.

The full output I'm getting in my test case is:

```javascript
System.register(["bluebird"], function (_export) {
  var _bluebird, foo;

  function* () {
    yield Q;
  }

  return {
    setters: [function (_bluebird2) {
      _bluebird = _bluebird2;
    }],
    execute: function () {
      "use strict";

      foo = _bluebird.coroutine();

      _export("foo", foo);
    }
  };
});
```

Which also seems to be an invalid call as well to the coroutine.

**Comment by guybedford** [comment]
:+1: 

Thanks so much - I'm jealous of your issue queue right now :)

**Comment by sebmck** [comment]
Whoops, forgot to say this was fixed as of 5.1.0. :runner: :dash:
